web/src/common/api.service.js

```diff
@@ -5,7 +5,7 @@ import VueAxios from 'vue-axios';
 const ApiService = {
   init() {
     Vue.use(VueAxios, axios);
-    Vue.axios.defaults.baseURL = '/api/v1';
+    Vue.axios.defaults.baseURL = `${process.env.VUE_APP_SERVER_ADDRESS}/api/v1`;
```

Comment:
Maybe set this in `main.js` and pass it through the init function so that it's easier to find?  Someone looking for the API base url config would start there.

Could you also use `(new URL('/api/v1', process.env.VUE_APP_SERVER_ADDRESS)).href` ?

It handles issues where a trailing slash conflicts with a slash prefix, and throws an exception if the url is invalid.  https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

It doesn't accept an empty string or slash to indicate same origin, but I don't think that case applies anymore anyway.  if needed, `window.location.href` can be input as the base url (and it will be trimmed appropriately).